### PR TITLE
Solves the layout shifting problem. 

### PR DESCRIPTION
**Pull request by Gitstar-OC**
Solves #904

### CODE REVIEW

**Comment by Gitstar-OC** [comment]
Wait, don't merge this for now.

**Comment by Gitstar-OC** [comment]
You can merge this, #898 will add the projects section (this get's that) back.

File: src/app/marketing/page.tsx [hunk 1]
```diff
@@ -826,6 +826,8 @@ export default async function Marketing() {
         </div>
       </div>
 
+      {/* <Projects /> */}
```
Comment (malted) [inline src/app/marketing/page.tsx:829]: Seems like you're including `/public/dummyImage.png`, `/src/app/marketing/components/Projects.jsx` in this PR accidentally - can you remove them?

**Review by malted — DISMISSED** [review]

**Comment by Gitstar-OC** [comment]
@malted Ready for merge :)

**Comment by taciturnaxolotl** [comment]
seems good to me!

**Pull request merged**